extensions/roulette/unmute/cog.py

```diff
@@ -0,0 +1,88 @@
+import logging
+
+from . import action
+from ..config import config
+from ..config.redis_config import redis_client
+
+from datetime import datetime
+from discord import Role
+from discord.ext import tasks
+from discord.ext.commands import Bot, Cog
+from typing import Set
+
+
+class Unmute(Cog):
+    def __init__(self, bot: Bot):
+        self.bot = bot
+        self.logger = logging.getLogger("roulette.unmute")
+        self.unmute_loop.change_interval(minutes=config.unmute_rate())
+        self.unmute_loop.start()
+        self.logger.info("Loaded Unmute cog")
+
+    async def cog_command_error(self, ctx, error: Exception) -> None:
+        self.logger.warning(str(error))
+
+    @tasks.loop(minutes=1)
```

Comment:
Just curious: can this not be set directly from config? (I would guess so since it's a decorator function).

If so: If possible, set this to `sys.MAXSIZE` instead - Redis requests (depending on deployment situation) can be charged per command, so we don't to send any more requests than necessary until the config is loaded.

Reply:
If you mean the @tasks.loop(), it's the amount of time it waits between loops.

Reply:
Yeah, the value should preferably be set to `sys.MAXSIZE` here first. Since Redis requests can cost money, we want to minimise the number of calls until the config is loaded (or if there's an unexpected error in loading)